doc/stand-alone.md

```diff
@@ -228,3 +228,70 @@ ssl:
             - "yourdomain.com"
 #END
 ```
+
+#### How to config global rule
+
+```yaml
+global_rules:
+    -
+        id: 1
+        plugins:
+            response-rewrite:
+                body: "hello\n"
+```
+
+#### How to config consumer
+
+```yaml
+consumers:
+  - username: jwt
+    plugins:
+        jwt-auth:
+            key: user-key
+            secret: my-secret-key
+```
+
+#### How to config plugin metadata
+
+```yaml
+upstreams:
+  - id: 1
+    nodes:
+      "127.0.0.1:1980": 1
+    type: roundrobin
+routes:
+  -
+    uri: /hello
+    upstream_id: 1
+    plugins:
+        http-logger:
+            batch_max_size: 1
+            uri: http://127.0.0.1:1980/log
+plugin_metadata:
+  - id: http-logger # note the id is the plugin name
```

Comment:
Do we have clear documentation about this tip?

Reply:
@juzhiyuan 
This one.

Reply:
I find it here, why not use `plugin_name` like here[1]?

[1] https://github.com/apache/apisix/blob/master/doc/admin-api.md#plugin-metadata